docs/user-guide/configuration.md

```diff
@@ -148,7 +148,7 @@ pages:
 ```
 
 See the section on [configuring pages and
-navigation](/user-guide/writing-your-docs.md#configure-pages-and-navigation) for
+navigation](/writing-your-docs.md#configure-pages-and-navigation) for
```

Comment:
Relative links don't start with a `/`. This is a valid change, but let's do it correctly: `(writing-your-docs.md#configure-pages-and-navigation)`